### PR TITLE
Add code for handling widget swapping between single/multi pane view

### PR DESCRIPTION
**Pull request by kevinwlau**
Able to switch widgets between single and multi pane view. Initialization works in both single and multi pane view.

`step-content.js` needs to be updated to load the widgets in the single column view, currently only puts widgets in the right-side pane which is invisible in the multi pane view.

**To Enable**: In a guide's main html, add `- interactive-guides/iguide-multipane` under the js imports.
Issue for enabling in circuit breaker guide: OpenLiberty/iguide-circuit-breaker#134

### CODE REVIEW

File: js/interactive-guides/iguide-multipane.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+    //TODO: when switching to multi, determine what step is selected 
+    // and show correct widgets (iguides-common PR #185)
+    $(window).on('resize', function() {
+        if (currentView == 'multi' && window.innerWidth <= 1170) {
+            // move widgets to single column
+            currentView = 'single';
+            console.log('Switch to single column at ', window.innerWidth);
+
+            var widgetDivsArray = [].slice.call( widgetDivs.detach() );
```
Comment (kevinwlau) [inline js/interactive-guides/iguide-multipane.js]: In this line, I use a quick hack to convert the NodeList returned by JQuery's `detach()` function into a standard array.

File: js/interactive-guides/iguide-multipane.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
+var currentView;
+var widgetDivs;
+function init() {
+    if (document.body.scrollWidth <= 1170) {
```
[the remark this answers is not in the document]
Comment (kevinwlau) [reply js/interactive-guides/iguide-multipane.js]: Ah, I missed this one when changing them to `window.innerWidth`

File: js/interactive-guides/iguide-multipane.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+        if (currentView == 'multi' && window.innerWidth <= 1170) {
+            // move widgets to single column
+            currentView = 'single';
+            console.log('Switch to single column at ', window.innerWidth);
+
+            var widgetDivsArray = [].slice.call( widgetDivs.detach() );
+            widgetDivsArray.map(widget => {
+                var step = widget.dataset.step;
+                //alternatively, $("#contentContainer .sect1#" + step + "_content")[0];
+                var contentStepDiv = $("#contentContainer .sect1:has(div[data-step='" + step + "'])")[0];
```
[the remark this answers is not in the document]
Comment (kevinwlau) [reply js/interactive-guides/iguide-multipane.js]: Changed logic to append as a child under the correct section. This doesn't prevent the `.sect1` widgets from appearing below `.sect2`.

Added css to hide the `.disableContainer` class in single column view. 
This also visually solves the above problem with the `.sect2` classes as none of the guides have widgets in both the main and subsection of one step.

**Comment by kevinwlau** [comment]
Updated #162 with instructions on enabling this file in other guides.

OpenLiberty/iguide-circuit-breaker#134 is the PR for enabling the changes.

File: js/interactive-guides/iguide-multipane.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (dmorgant) [reply js/interactive-guides/iguide-multipane.js]: I think you still need to account for embedded .sect2 divs within the .sect1s.  Future guides may have widgets in the sect1s that have an embedded sect2, or may have widgets in both.  Seems to me you should be able to add a search here like
contentStepDiv.find('.sect2')
under line 40ish of iguide-multipane.js
var contentStepDiv = $('#contentContainer #' + step + '_content')[0]; 
and if the return indicates there are sect2 divs within the section then insert the .stepWidgetContainer above the sect2 div.
